app/assets/javascripts/cart.js.coffee

```diff
@@ -70,32 +70,27 @@ $(document).ready ->
     srid = $(this).data('srid')
     editing_ssr = $(this).data('editing-ssr')
     li_count = parseInt($('#line_item_count').val())
-    has_fulfillments = $(this).data('has-fulfillments')
     request_submitted = $(this).data('request-submitted')
     spinner = $('<span class="spinner"><img src="/assets/catalog_manager/spinner_small.gif"/></span>')
 
-    if has_fulfillments == 1
-      $('#modal_place').html($('#has-fulfillments-modal').html())
```

Comment:
Since this is gone, can you remove app/views/service_requests/modals/_has_fulfillments_modal.html.haml